util/kubernetes/client/templates.go

```diff
@@ -84,7 +84,7 @@ spec:
           - {{.}}
           {{- end }}
           image: {{ .Image }}
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Not 100% sure how `micro update` works but would this break that? I suspect not because I think the image pulls down a binary rather than having a prebaked image for a service right?

Reply:
Now we have a build service this isn't an issue, it's the same cell every time. For micro/platform image we always use tags.

Reply:
It was taking 30s longer than needed to start a service for a user. I think it's better to have this as default and then override using an option if we need to.